client/my-sites/google-my-business/new-account/style.scss

```diff
@@ -0,0 +1,37 @@
+.new-account__card {
```

Comment:
My concern is that we could end up with another component called `new-account` elsewhere and these names could collide. What about `gmb-new-account`?

Reply:
I'll make that change, and also fix styles from the `SelectBusinessType` component.